### PR TITLE
Supplement Map maker's README.md

### PR DESCRIPTION
**Pull request by JostP**
Open to suggestions.

### CODE REVIEW

File: mods/ctf/ctf_map/map_maker/README.md [hunk 1]
```diff
@@ -89,3 +89,22 @@ treasures = default:pick_steel,0.5,5,1,10;shooter:shotgun,0.04,2,1;shooter:grena
 ```
 
 (See [here](../../../other/treasurer/README.md) to understand the magic numbers)
+
+### `screenshot`
+
+Every map must have its own screenshot in map's folder. It should have ratio around 3:2 (screenshot 600x400 is suggested).
```
Comment (Thomas--S) [inline mods/ctf/ctf_map/map_maker/README.md]:
```suggestion
Every map must have its own screenshot in map's folder. It should have an aspect ratio of 3:2 (screenshot 600x400px is suggested).
```

File: mods/ctf/ctf_map/map_maker/README.md [hunk 2]
```diff
@@ -89,3 +89,22 @@ treasures = default:pick_steel,0.5,5,1,10;shooter:shotgun,0.04,2,1;shooter:grena
...
+Six images which should be in map's folder.
+
+* `skybox_1.png` - up
+* `skybox_2.png` - down
+* `skybox_3.png` - east
+* `skybox_4.png` - west
+* `skybox_5.png` - south
+* `skybox_6.png` - north
+
+(You can find some good skyboxes with suitable licenses [here](https://opengameart.org/art-search-advanced?field_art_tags_tid=skybox).)
```
Comment (Thomas--S) [inline mods/ctf/ctf_map/map_maker/README.md]:
```suggestion
(You can find some good skyboxes with suitable licenses at [https://opengameart.org](https://opengameart.org/art-search-advanced?field_art_tags_tid=skybox).)
```

File: mods/ctf/ctf_map/map_maker/README.md (hunk 1 above)
Comment (JostP) [reply mods/ctf/ctf_map/map_maker/README.md]: :+1:

**Comment by JostP** [comment]
Also added license description and how to edit exported maps.

File: mods/ctf/ctf_map/map_maker/README.md [hunk 3]
```diff
@@ -89,3 +104,52 @@ treasures = default:pick_steel,0.5,5,1,10;shooter:shotgun,0.04,2,1;shooter:grena
...
+
+
+
+
+
+
+
+
+
+
```
Comment (ClobberXD) [inline mods/ctf/ctf_map/map_maker/README.md]: Why so many empty lines? :O

Comment (JostP) [reply mods/ctf/ctf_map/map_maker/README.md]: o_O
Fixing...

Comment (JostP) [reply mods/ctf/ctf_map/map_maker/README.md]: Done

**Comment by JostP** [comment]
If you want to test this go to JostP/capturetheflag/readme_skybox branch

**Comment by MinetestSam** [comment]
I think it is better to also a recommend licence for skyboxes

**Comment by JostP** [comment]
CC0, CC BY 3.0, CC BY 4.0, CC BY-SA 3.0, CC BY-SA 4.0. Anything else?

File: mods/ctf/ctf_map/map_maker/README.md [hunk 4]
```diff
@@ -78,6 +78,22 @@ format. Refer to the `treasures` sub-section for more details.
...
+```lua
+license = <name>
+```
+* If license require something (for example if you modify other's map and you have to tell who is author of the original map) you have to include this in `license`.
+If you want to tell more infomation, you can use:
+```lua
+others = <description>
+```
+* If you don't know which license to use, [this list of CC licenses](https://creativecommons.org/use-remix/cc-licenses/) can help you.
+* Recommended license is `CC BY-SA 4.0`, a very good all-rounder license, which most of our maps use.
```
Comment (Thomas--S) [inline mods/ctf/ctf_map/map_maker/README.md]:
```suggestion
* We can only accept Free Software licenses, e.g.`CC BY-SA 4.0`.
```

File: mods/ctf/ctf_map/map_maker/README.md [hunk 5]
```diff
@@ -78,6 +78,22 @@ format. Refer to the `treasures` sub-section for more details.
...
+Unless you know what you're doing, we recommend choosing `CC BY-SA 4.0` and don't just use it without having read any information about it.
```
Comment (Thomas--S) [inline mods/ctf/ctf_map/map_maker/README.md]:
```suggestion
* Please know what you are doing when choosing a certain license. For example, you can read information about various licenses and/or consult a lawyer.
```

File: mods/ctf/ctf_map/map_maker/README.md (hunk 4 above)
Comment (ClobberXD) [reply mods/ctf/ctf_map/map_maker/README.md]: Even CC BY-NC is a "free" license. It's open-source licenses that we're looking for, not free ones. :)

Comment (Thomas--S) [reply mods/ctf/ctf_map/map_maker/README.md]: No, CC BY-NC is not a Free license: https://www.gnu.org/licenses/license-list.html#CC-BY-NC

(Please also not the capitalized "F": It implies that not "free" as in _free beer_ is meant, but as in _Free Speech_.

Please read about the difference of Open Source and Free Software: https://www.gnu.org/philosophy/open-source-misses-the-point.en.html

Comment (ClobberXD) [reply mods/ctf/ctf_map/map_maker/README.md]: Ah, a capitalised `F` means a completely different thing. Kewl...

File: mods/ctf/ctf_map/map_maker/README.md [hunk 6]
```diff
@@ -78,6 +78,22 @@ format. Refer to the `treasures` sub-section for more details.
 * `phys_jump`: [Optional] Player jump multiplier. Accepts any valid number. Defaults to 1.
 * `phys_gravity`: [Optional] Player gravity multiplier. Accepts any valid number. Defaults to 1.
 
+#### `license`
+
+* Every map must have its own license. Once you've chosen your license, simply add the following line to the map's `.conf` file:
+```lua
```
Comment (rubenwardy) [inline mods/ctf/ctf_map/map_maker/README.md]: code blocks in bullet lists should be indented so that they are part of the lists, and also have blank lines before and after

Comment (JostP) [reply mods/ctf/ctf_map/map_maker/README.md]: Good? 👇

Comment (ClobberXD) [reply mods/ctf/ctf_map/map_maker/README.md]: No, the backticks themselves have to be indented by two spaces, along with the contents of the code block.

It seems like you've also indented the contents of code-blocks with tabs.

Comment (JostP) [reply mods/ctf/ctf_map/map_maker/README.md]: What about now?

File: mods/ctf/ctf_map/map_maker/README.md [hunk 7]
```diff
@@ -82,20 +82,20 @@ format. Refer to the `treasures` sub-section for more details.
 
 * Every map must have its own license. Once you've chosen your license, simply add the following line to the map's `.conf` file:
 
-```lua
+  ```lua
```
Comment (ClobberXD) [inline mods/ctf/ctf_map/map_maker/README.md]: These empty lines within the code blocks aren't required

Comment (JostP) [reply mods/ctf/ctf_map/map_maker/README.md]: Ok, fixing

Comment (JostP) [reply mods/ctf/ctf_map/map_maker/README.md]: fixed

File: mods/ctf/ctf_map/map_maker/README.md [hunk 8]
```diff
@@ -78,14 +78,69 @@ format. Refer to the `treasures` sub-section for more details.
...
 `treasures` is a list of treasures to be registered for this map in serialized format.
 
 An example `treasures` value that registers steel pick, shotgun, and grenade:
 
-```lua
-treasures = default:pick_steel,0.5,5,1,10;shooter:shotgun,0.04,2,1;shooter:grenade,0.08,2,1
-```
+  ```lua
+  treasures = default:pick_steel,0.5,5,1,10;shooter:shotgun,0.04,2,1;shooter:grenade,0.08,2,1
+  ```
```
Comment (ClobberXD) [inline mods/ctf/ctf_map/map_maker/README.md:109]: This code-block isn't a part of a list. Please revert the changes to these lines.

File: mods/ctf/ctf_map/map_maker/README.md [hunk 9]
```diff
@@ -78,14 +78,69 @@ format. Refer to the `treasures` sub-section for more details.
...
+* `skybox_1.png` - up
+* `skybox_2.png` - down
+* `skybox_3.png` - east
+* `skybox_4.png` - west
+* `skybox_5.png` - south
+* `skybox_6.png` - north
+
+You have to include skybox license in `license` in `.conf` file. We can only accept Free Software licenses, e.g. `CC0`, `CC BY 3.0`, `CC BY 4.0`, `CC BY-SA 3.0`, `CC BY-SA 4.0`.
+
+(You can find some good skyboxes with suitable licenses at [https://opengameart.org](https://opengameart.org/art-search-advanced?field_art_tags_tid=skybox).)
```
Comment (ClobberXD) [inline mods/ctf/ctf_map/map_maker/README.md:132]: I think the outer pair of parenthesis isn't required. i.e. `(You can find some ... )` -> `You can find some ... .`

File: mods/ctf/ctf_map/map_maker/README.md [hunk 10]
```diff
@@ -78,14 +78,69 @@ format. Refer to the `treasures` sub-section for more details.
...
 
+#### `license`
+
+* Every map must have its own license. Once you've chosen your license, simply add the following line to the map's `.conf` file:
+
+  ```lua
+  license = <name>
+  ```
+
+* If license require something (for example if you modify other's map and you have to tell who is author of the original map) you have to include this in `license`.
```
Comment (ClobberXD) [inline mods/ctf/ctf_map/map_maker/README.md:89]: Slightly improved wording; hope you don't mind.

```suggestion
* If attribution is required (for example if you modify other's map and you have to tell who is author of the original map), that has to be appended to the `license` field.
```

**Comment by JostP** [comment]
Should be ready for approving now
